1_data/cfg/dates.yml

```diff
@@ -0,0 +1,6 @@
+calibrate:
+  start: 2012-05-01
+  end: 2018-03-01
```

Comment:
Do we want calibration and forecast times to overlap? Could end calibration time at beginning of forecast time

Reply:
Sure. Just cut out calibrate$end field, you think?